src/common/src/common/models.py

```diff
@@ -0,0 +1,15 @@
+import json
+from typing import Tuple, Any
+
+
+class LogEntry:
+    def __init__(self, time: str, pos: Tuple[float, float], steering: float, throttle: float, has_collided: bool):
+        self.time = time
+        self.pos = pos
+        self.steering = steering
+        self.throttle = throttle
+        self.has_collided = has_collided
+
+    def default(self, o: Any) -> Any:
+        return json.dumps(dict(o))
```

Comment:
What does this function do?

Reply:
That's from me testing different serialization methods. I'll test it without later and remove it of it's unnecessary

Reply:
Alright, sounds good then